cylc/uiserver/resolvers.py

```diff
@@ -397,10 +472,31 @@ async def service(
                 executor=self.executor
             )
 
-        else:
+        elif command == 'play':
             return await Services.play(
                 workflows,
                 kwargs,
                 self.workflows_mgr,
                 log=self.log
             )
+
+        raise Exception()
```

Comment:
Might be worth using `NotImplementedError` here as it's a little more descriptive than a blank exception and will get excluded from coverage.

```suggestion
        raise NotImplementedError()
```